prov/psm2/src/psmx2_rma.c

```diff
@@ -900,6 +904,7 @@ ssize_t psmx2_write_generic(struct fid_ep *ep, const void *buf, size_t len,
 	void *psm2_context;
 	int no_event;
 	int err;
+static unsigned write_count = 0;
```

Comment:
nit: indentation

Reply:
oops, that's a left over test code. should be removed.